iroha-lib/README.md

```diff
@@ -1,16 +1,113 @@
-# C++ library for Hyperledger Iroha
+# Hyperledger Iroha C++ library
 
 
-This is a source code for HL Iroha C++ library.
+### Current version of the library was tested and compatible with Iroha 1.5.0
 
-Currently, latest HL Iroha 1.5 release (`hyperledger/iroha:latest` Docker image) is supported.
+The library was created to provide a convenient interface for C++ applications to communicate with [Iroha](https://github.com/hyperledger/iroha) blockchain including sending transactions/query, streaming transaction statuses and block commits.
 
 
-### Installation
+# iroha-lib
 
-```bash
-mkdir build && cd build && cmake .. && make 
+Client library of [Iroha](https://github.com/hyperledger/iroha) written completely in modern C++.
+Currently, the latest HL Iroha 1.5 release (`hyperledger/iroha:latest` Docker image) is supported.
```

Comment:
I don't know if soon `hyperledger/iroha:latest` will be still for Iroha 1, so I suggest:
```suggestion
Currently, the latest HL Iroha 1.5 release (`hyperledger/iroha:1.5.0` Docker image) is supported.
```

Reply:
Don't worry, we'll make the necessary updates in either case.